app/models/question_answer_comment.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+class QuestionAnswerComment < ActiveRecord::Base
+  include Trashable
+
+  # Bump this when changing MARKDOWN_FEATURES or MARKDOWN_IT_RULES
+  COOKED_VERSION = 1
+
+  belongs_to :topic
```

Comment:
Actually thinking about it, is there a need for a comment to be tied to a topic? Since a post is already tied to a topic.

Reply:
Actually let me remove this. I added this initially as a potential performance optimization where we might want to filter for comments in a topic. Thinking about it now, I can just join against the posts table.